src/south/ProtocolHandler.class.js

```diff
@@ -174,6 +180,19 @@ class ProtocolHandler {
       setTimeout(resolve, timeout)
     })
   }
+
+  /**
+   * Get live status.
+   * @returns {object} - The live status
+   */
+  getStatus() {
+    const status = { 'Last scan time': this.lastOnScanAt ? new Date(this.lastOnScanAt).toLocaleString() : 'Never' }
+    if (this.handlesFiles) {
+      status['Last file added time'] = this.lastAddFileAt ? new Date(this.lastAddFileAt).toLocaleString() : 'Never'
+      status['Number of files added'] = this.addFileCount
+    }
```

Comment:
need to also track Last value and number of values, last timestamp, last pointid,

Reply:
I was planning to include point related informations when we implement it for the first point compatible South.

Reply:
I think the default status could be added now as well.